src/test/java/org/broadinstitute/hellbender/engine/FeatureSupportInVariantsWalkersIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.examples.ExampleVariantWalker;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public final class FeatureSupportInVariantsWalkersIntegrationTest extends CommandLineProgramTest {
+
+    @Override
+    public String getTestedClassName() {
+        return ExampleVariantWalker.class.getSimpleName();
+    }
+
+    @Test(expectedExceptions = UserException.class)
+    public void testCountVariants_requiresIndexForInterval() throws Exception {
```

Comment:
These test methods all have `CountVariants` in their names, even though they are no longer running `CountVariants`